tensorflow/core/lib/strings/str_util.h

```diff
@@ -131,6 +133,18 @@ std::vector<string> Split(StringPiece text, StringPiece delims);
 template <typename Predicate>
 std::vector<string> Split(StringPiece text, StringPiece delims, Predicate p);
 
+// Split "text" into into a vector of strings where each string
+// in the array is a string of ASII or unicode chars. The `delim`
+// is a string with 0 or 1 chars.
```

Comment:
Should be stay "`delim` is a single UTF-8-encoded unicode character or the empty string" (to avoid any confusion, since `delim.size()` can be more than 1)?

Reply:
Done.